requirements.txt

```diff
@@ -4,5 +4,7 @@ sh
 # Testing
 pytest<5.0.0,>=3.3.0
 pytest-cookies
+pytest-cov
```

Comment:
Don't think we need that dependency in the template -- we need it in the generated package